Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
```

Comment:
Now that I'm thinking about how this script actually integrates into the build, you should not be calling this script. Instead, you should take the necessary parameters as arguments to the script, so that the csproj can pass them in.

Reply:
Updated the script to take arguments from GVFS.Installer.Mac.csproj.